src/components/popover/popover.component.ts

```diff
@@ -31,49 +34,95 @@ enum TabName {
     selector: 'popover',
     template: require('./popover.component.html'),
     styles: [require('./popover.component.scss')],
-    host: { 'class': 'popover' }
+    animations: [
+        trigger('popover', [
+            state('closed', style({
+                transform: 'translateY(30px)',
+                visibility: 'hidden',
+                opacity: 0
+            })),
+            state('opened', style({
+                transform: 'translateY(0)',
+                visibility: 'visible',
+                opacity: 1
+            })),
+            transition('opened => closed', [
+                animate('200ms ease-out', keyframes([
+                    style({ transform: 'translateY(0)', visibility: 'visible', opacity: 1, offset: 0 }),
+                    style({ transform: 'translateY(30px)', visibility: 'hidden', opacity: 0 , offset: 1 })
+                ]))
+            ]),
+            transition('closed => opened', [
+                animate('200ms ease-out', keyframes([
+                    style({ transform: 'translateY(30px)', visibility: 'visible', opacity: 0, offset: 0 }),
```

Comment:
The visibility should be in the style of the transition, not in the style of the animate, since it is not something we want to be animated.